sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -397,6 +393,7 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]] extends Product {
   def argString: String = productIterator.flatMap {
     case tn: TreeNode[_] if containsChild(tn) => Nil
     case tn: TreeNode[_] => s"${tn.simpleString}" :: Nil
+    case seq: Seq[_] if seq == Seq.empty => "[Empty]" :: Nil
```

Comment:
I think it's used to fix this line:
`case seq: Seq[BaseType] if seq.toSet.subsetOf(children.toSet) => Nil`

We can't reflect on the parameter type, so empty Seq will go into this branch and return nothing.